client/src/details/details.cpp

```diff
@@ -473,3 +483,41 @@ void Details::setItemsTreeModel(ItemsTreeModel *model)
     // keep the member - could be useful later
     mItemsTreeModel = model;
 }
+
+void Details::copyAddressFromGroup(QGroupBox *box)
+{
+    QMap<QString, QString> fieldNames;
+    fieldNames.insert(QStringLiteral("city"), {});
```

Comment:
These 5 inserts serve no purpose, fillAddressFieldsMap will take care of filling the map anyway.

Sorry for missing this in the first round of review, initially I didn't spot the fact that the fillAddressFieldsMap call was going to fill the same map.

Reply:
Yes, My idea was to have a fixed Map keys and allow the 'fillAddressFieldsMap' only write the values, but I could not find a way to do that. (A extra class will work, but I do not think that is worth).

Removed